package.json

```diff
@@ -106,7 +105,8 @@
     "//2": "(E.g. see https://github.com/gulpjs/gulp/issues/2162 and https://github.com/nodejs/node/issues/25132.)",
     "natives": "1.1.6",
     "//3": "`graceful-fs` needs to be pinned to support gulp 3, on Node v12+",
-    "graceful-fs": "^4.2.3"
+    "graceful-fs": "^4.2.3",
+    "marked": "0.3.6"
```

Comment:
Rather than pinning `marked` we should just fix the docs to be CommonMark compliant.

Reply:
OK, I wasn't sure that was something that we were open to doing. I'll look into it.